lib/auth/userloginstate/generator.go

```diff
@@ -186,8 +197,6 @@ func (g *Generator) Generate(ctx context.Context, user types.User, ulsService se
 	return uls, nil
 }
 
-// addAccessListsToState will add the user's applicable access lists to the user login state,
-// returning any inherited roles and traits.
```

Comment:
Why did we remove the godoc here? If it's no longer accurate can we replace it with something that is accurate rather than removing it completely?

Also, while you're here, it probably makes sense to name the return values - it's not clear from the name of the function what the `[]string` and `map[string][]string` are supposed to be.